File: Day-11/Q_42.cpp
//Write a program to Write function to find  maximum

#include<iostream>
#include<math.h>
using namespace std;

int max(int a,int b,int c){
    int num_max=max(a,max(b,c));
    return num_max;
}
 int main(){
    int num1,num2,num3;
    cout<<"Enter 3 numbers: ";
    cin>>num1>>num2>>num3;
    cout<<"Largest number is: "<<max(num1,num2,num3);
}

File: Day 11/Q42.java
import java.util.Scanner;
public class Q42
{
    public static int maximum(int n1,int n2)
    {
        if(n1>n2)
        {
            return n1;
        }
        return n2;
    }
    public static void main()
    {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter first number = ");
        int num1=sc.nextInt();
        System.out.print("Enter second number = ");
        int num2=sc.nextInt();
        int max=maximum(num1, num2);
        System.out.println("Maximum value is: "+max);
    }
}
